enki/PhysicalEngine.cpp

```diff
@@ -225,14 +225,15 @@ namespace Enki
 	
 	const double PhysicalObject::g = 9.81;
 	
-	PhysicalObject::PhysicalObject(void) :
+	PhysicalObject::PhysicalObject(bool mouvable) :
```

Comment:
For me it's necessary because the movability is an attribute of each object, used only by the viewer but also like the led color ...
And specify it in constructor permit to define robot movable or not in the playground XML definition. For example the epuck robot in the R2T2 simulation.